app-dicts/myspell-sv/Manifest

```diff
@@ -1,3 +1,6 @@
+DIST hyph-sv.oxt 51981 BLAKE2B b2b6f1c913019795da532115b04947f143d78fcc9e0e8ee3f12a65841281900a3a8c24e158fa9122da9ce64af3f4d6ed8d0efa02143cb3aeb2db0ec0f011a292 SHA512 2dea472c191dc1992eaf28718e575f9b07526e0acdf511f8359588b118c7fc2578f8b5b53d23eeda34f1391807546ff5a0ba6de088c42588ec94daca4232cf7a
 DIST myspell-sv-2.14-hyph.oxt 44382 BLAKE2B 2f25cd604c79d51549d47110d06e8877431bd9be521a57e7eeb172817d32b2f11b07e6c1c212c4f6f9d49ce7a4d3b54ecdb18bbab41a7cd26bc3c8de9a553fc1 SHA512 0f71809d2d401bb917037bb50a2fa88ffe825cac081cb14e9378ffeebc447144b733a9d4fd99cc5bc1f1a37835cc880210512ccc79341a4ba8c6cb5dcc38fc40
 DIST myspell-sv-2.14-thes.oxt 306542 BLAKE2B 0a5ebf5577e6fb8c492b13529ea904d1aceded3814e97f79e8581c55122cc647e4a32ce2bba148b123f97612aef156c5b6f425e8940d1cf80b808c259eef6fd5 SHA512 e637dc92bfc9bc86ccf10710fac1b036a99b0abc2888d30ea7dbdaca1c3cbec57fd7a184648d922b120a26f7c151b7cc043f783fb35b7e121ae22458f0a72103
+DIST ooo-swedish-dict-2-42.oxt 1466823 BLAKE2B 8be0ab5be29dc3d4e3316732992e2a6f37f50fad1035f0913cb069e33fe7c1b453cdb814701512ba75029592dfc7d2548a341018bb7fd3319ce571bc042c4135 SHA512 9aab2a1d0def637195c676e93712c84391f78fd14fefd8a0734099a3957cf99b9b3e0d41bf9cbaf7b6cb6c6aa5de58bc92169b844f0c69343b87483b324423e0
 DIST ooo_swedish_dict_2.14.oxt 1288382 BLAKE2B 54a46792386e794dc836e0178929cdad75e29007ac981aaeeeb74da996d765ff8d737796568576662431c95899824d6460f099c6814fed673844ba295967627b SHA512 77ab715ccb66c7e14ed84189dd0847c809824a3cf0881ee39d08f6d4ef89eccfeafcdbde1759bfa8224650b1907760d9a48f40e464211ae12593ed5525d6372b
+DIST swedishthesaurus.oxt 306542 BLAKE2B 0a5ebf5577e6fb8c492b13529ea904d1aceded3814e97f79e8581c55122cc647e4a32ce2bba148b123f97612aef156c5b6f425e8940d1cf80b808c259eef6fd5 SHA512 e637dc92bfc9bc86ccf10710fac1b036a99b0abc2888d30ea7dbdaca1c3cbec57fd7a184648d922b120a26f7c151b7cc043f783fb35b7e121ae22458f0a72103
```

Comment:
Again, upstream has versioned these, but you have not.

Reply:
Done